examples/nl-normal-address-maps.html

```diff
@@ -155,7 +153,7 @@
 	</div>
 </div>
 
-<script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAyBuzlPSNhmRIEhIl-3ZUidj3fwXfsDSw&amp;sensor=false"></script>
+<script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?key=API_KEY&amp;sensor=false"></script>
```

Comment:
Alleen dit script inladen als het echt nodig is

Reply:
Done!